client/src/modules/purchases/create/create.html

```diff
@@ -134,23 +134,13 @@
         </div>
 
         <!-- "Add number of grid rows" input-group -->
-        <div class="input-group">
-          <span class="input-group-btn">
-            <button
-              id="btn-add-rows"
-              class="btn btn-default"
-              ng-disabled="!PurchaseCtrl.supplier"
-              ng-click="PurchaseCtrl.addItems(PurchaseCtrl.itemIncrement)">
-              <span class="fa fa-plus-circle"></span> {{ "FORM.BUTTONS.ADD" | translate }}
-            </button>
-          </span>
-          <input
-            type="number"
-            class="form-control"
-            ng-model="PurchaseCtrl.itemIncrement"
-            style="max-width : 40px;"
-            ng-disabled="!PurchaseCtrl.supplier">
-          </div>
+        <bh-add-item
+          item-increment = "PurchaseCtrl.itemIncrement"
```

Comment:
In my view, the component should handle this itself - there is no reason for the parent controller to care about the item increment.  That will clean up the component bindings and the parent controllers significantly.